src/DFTK.jl

```diff
@@ -32,6 +32,7 @@ include("common/threading.jl")
 include("common/printing.jl")
 include("common/cis2pi.jl")
 include("common/zeros_like.jl")
+include("common/phonon.jl")
```

Comment:
I don't like that name, because no-one knows that would be in there. EIther you rename the file to sth like `norm.jl` or you just put the function definition in a `postprocess/phonon.jl` that you will later extend.